### PR TITLE
Added Wiki.js Template

### PR DESCRIPTION
**Pull request by Ponkhy**

### CODE REVIEW

**Comment by deiucanta** [comment]
Should we add SQLite as a DB option?

**Comment by Ponkhy** [comment]
Would do the same as mentioned [here](https://github.com/easypanel-io/templates/pull/15#issuecomment-1124875239).

**Comment by Ponkhy** [comment]
Looks like there is a problem with SQLite and Wiki.js right now.
The beta of Wiki.js 3 also came out this month and will support only Postres anyway.
So i'd say we stay with the current config.

File: templates/wikijs.ts [hunk 1]
```diff
@@ -0,0 +1,98 @@
...
+        port: 3000,
+        secure: true,
+      },
+      domains: [{ name: domain }],
+    };
+
+    const postgresService: PostgresService = {
+      projectName,
+      serviceName: databaseServiceName,
+      image: "postgres:alpine",
```
Comment (deiucanta) [inline templates/wikijs.ts:79]: do you need to change the default image for postgres?

File: templates/wikijs.ts [hunk 2]
```diff
@@ -0,0 +1,98 @@
...
+        title: "App Service Name",
+        default: "wikijs",
+      },
+      databaseType: {
+        type: "string",
+        title: "Database Type",
+        oneOf: [
+          { enum: ["postgres"], title: "Postgres" },
+          { enum: ["mysql"], title: "MySQL" },
+        ],
```
Comment (deiucanta) [inline templates/wikijs.ts:34]: you can add a `default` prop for `postgres`

**Comment by deiucanta** [comment]
> Looks like there is a problem with SQLite and Wiki.js right now. The beta of Wiki.js 3 also came out this month and will support only Postres anyway. So i'd say we stay with the current config.

Sounds good.

File: templates/wikijs.ts [hunk 3]
```diff
@@ -0,0 +1,106 @@
...
+    appServiceName,
+    databaseType,
+    databaseServiceName,
+  }) {
+    const services: Services = [];
+    const databasePassword = randomPassword();
+    const databaseUsername = databaseType === "postgres" ? "postgres" : "mysql";
+    const databasePort = databaseType === "postgres" ? "5432" : "3306";
+
+    services.push({
```
Comment (deiucanta) [inline templates/wikijs.ts:60]: I think you forgot to specify the service type

Comment (Ponkhy) [reply templates/wikijs.ts:60]: Haha, Thanks, was a little too quick there.

**Pull request merged**